stdlib/smtpd.pyi

```diff
@@ -81,7 +82,8 @@ class PureProxy(SMTPServer):
         self, peer: _Address, mailfrom: str, rcpttos: list[str], data: bytes | str
     ) -> str | None: ...
 
-class MailmanProxy(PureProxy):
-    def process_message(  # type: ignore
-        self, peer: _Address, mailfrom: str, rcpttos: list[str], data: bytes | str
-    ) -> str | None: ...
+if sys.version_info < (3, 11):
+    class MailmanProxy(PureProxy):
```

Comment:
The whole `smtpd.py` seems to be gone, so the correct fix would be to edit `stdlib/VERSIONS`.

Reply:
The wholesale removal of `smtpd` may be reverted; we're waiting on a decision from the Steering Council on that one. python/steering-council#86. I'll just take the changes to the file out for now, since there's not much point adding the if `sys.version_info` check if the whole module is removed.